days-since-start-of-the-year.js

```diff
@@ -0,0 +1,12 @@
+const now = new Date();
+const currentTimestamp = now.getTime();
+const daysSinceTheStartOfTheYear = () => {
+  const then = new Date("2021-01-01");
```

Comment:
It would be super nice if this would work next year as well 🙂

Hint: you can use [Date.setFullYear()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setFullYear)